operational/flexera/custom_branding/README.md

```diff
@@ -0,0 +1,75 @@
+
+### Category : MSP Specific Settings
+
+- *Login Logo Image* - The web address of the image resource file location to use in place of the Flexera logo for unauthenticated pages like login, password reset, and so on. Must start with http  :// or https  ://
+- *Global Customer Support Link* - A URL override pointing to your own Customer Support link on the login page and elsewhere throughout the app. Must start with http : // or https : //
+- *Marketing Login Image* - The web address of the image resource file location to use in place of the Flexera marketing section on the login page. Must start with http : // or https : //
+- *Marketing Login CTA Link* - A URL override pointing to your own Download Now call to action link on the login page. Must start with http : // or https : //
+- *Marketing Login CTA text* - Plain text override of the Download Now call to action text displayed on the login page.
+- *Marketing Login Eyebrow Text* - Plain text to override of the eyebrow marketing text on the login page.
+- *Marketing Login Blurb Text* - Plain text to override of the marketing blurb on the login page.
+- *Marketing Login Title Text* - Plain text override of the marketing title text on the login page.
+
+The [Partner MSP Cusotmizations](https://docs.flexera.com/flexera/EN/Administration/PartnerMSPCustomizations.htm) page in the docs has detailed information on the settings and diagrams for custom branding
+
+## Policy Actions
+
+The following policy actions are taken when branding fails.
+
+- Send an email report
+- Create an incident
+
+## Prerequisites
+
+This Policy Template uses [Credentials](https://docs.flexera.com/flexera/EN/Automation/ManagingCredentialsExternal.htm) for authenticating to datasources -- in order to apply this policy you must have a Credential registered in the system that is compatible with this policy. If there are no Credentials listed when you apply the policy, please contact your Flexera Org Admin and ask them to register a Credential that is compatible with this policy. The information below should be consulted when creating the credential(s).
+
+- [**Flexera Credential**](https://docs.flexera.com/flexera/EN/Automation/ProviderCredentials.htm) (*provider=flexera*) which has the following roles:
+  - `manage_organization`*
+
+  \* Required for taking branding actions.
+
+The [Provider-Specific Credentials](https://docs.flexera.com/flexera/EN/Automation/ProviderCredentials.htm) page in the docs has detailed instructions for setting up Credentials for the most common providers.
+
+- A sub domain need to be configured to ensure that the Branding of the logon screen takes effect as outlined in the [Partner MSP Customizations](https://docs.flexera.com/flexera/EN/Administration/PartnerMSPCustomizations.htm) documentation. Otherwise the policy will only allow tenant to update the landing page.

```

Comment:
This should not be a list item because this implies it is underneath "This Policy Template uses [Credentials]" when it's not a credential-related prerequisite at all. Remove the - and instead begin the sentence with "Additionally, " to indicate it is an additional requirement.

"subdomain" is a single word, not two.

"Branding" should not be capitalized.

"logon screen" should be replaced with "The Flexera login page" or something more clear.

"policy template", not "policy"